src/PolySharp.SourceGenerators/PolyfillsGenerator.TypeForwards.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections.Immutable;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using PolySharp.SourceGenerators.Extensions;
+using PolySharp.SourceGenerators.Helpers;
+
+namespace PolySharp.SourceGenerators;
+
+/// <inheritdoc/>
+partial class PolyfillsGenerator
+{
+    /// <summary>
+    /// The collection of fully qualified type names for types that could require a <c>modreq</c>.
+    /// </summary>
+    private static readonly ImmutableArray<string> ModreqCandidateFullyQualifiedTypeNames = ImmutableArray.Create(
+        "System.Index",
+        "System.Range",
+        "System.Runtime.CompilerServices.IsExternalInit");
```

Comment:
I would appreciate it very much if there was a way to easily control this list from a property (and to disable it altogether if desired), I've explicitly listed all types to include for my project by tfms and like being able to have this level of control. If the list is explicitly specified, ideally (for me), it would skip checking if the type exists and just write the code to forward it.

Reply:
Resolving this, as we said we can look into this separately if needed 🙂